core/src/main/java/org/apache/calcite/sql/SqlDialect.java

```diff
@@ -1445,16 +1448,54 @@ protected String getDateTimeFormatString(
     return new Pair<>(dateTimeTokens, separators);
   }
 
+  private Pair<List<String>, List<Character>> getDateTimeTokensIfNotFound(
+      String standardDateFormat) {
+    List<String> dateTimeTokens = new ArrayList<>();
+    List<Character> separators = new ArrayList<>();
+    standardDateFormat = standardDateFormat.replace("''","'" );
```

Comment:
As discussed remove this as we are already handling in BigQuerySqlDialect.
Also check in other places as well.

Reply:
done